src/app/query/balance/balance.hooks.ts

```diff
@@ -73,3 +78,14 @@ export function useAddressAnchoredAvailableStxBalance(address: string) {
     return balances.stx.balance.minus(balances.stx.locked);
   }, [balances]);
 }
+
+export function useAccountsAvailableStxBalance(accounts?: AccountWithAddress[]) {
+  const accountsBalances = useGetAnchoredAccountBalanceListQuery(accounts);
+
+  return useMemo(() => {
+    return accountsBalances.reduce((acc, balance) => {
+      return acc.plus(balance.data?.stx.balance || 0);
+    }, new BigNumber(0));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Summing logic could be made into a shared function maybe?

Reply:
I will address this once we get the release out ...not ignoring it!